src/components/Balance/Balance.module.css

```diff
@@ -0,0 +1,36 @@
+@import '../../utils/css-variables.css';
+.BalanceContainer {
+  width: 280px;
+  background-color: var(--color-bg-secondary);
+  border-radius: var(--radius-widget-binance);
+  height: 80;
+  text-transform: uppercase;
+  padding: 8px 32px 11px;
+}
+
+@media (min-width: 780px) {
+  .Balance {
+    width: 360px;
+    padding: 8px 40px 11px;
+  }
+}
+@media (min-width: 1200px) {
+  .Balance {
+    width: 395px;
+  }
+}
```

Comment:
Nie działa RWD, mamy tu dwóch winowajców:
W zapytaniach medialnych brakuje screen (@media screen and (....) {})
Jest podana klasa Balance, a powinna być BalanceContainer